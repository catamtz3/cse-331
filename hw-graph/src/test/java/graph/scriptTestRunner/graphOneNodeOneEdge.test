# Create a graph
CreateGraph oneNode

#Add node to the graph
addNode oneNode a

#Add edge to the graph
addEdge oneNode a a aa

# Get the nodes of the graph
ListNodes oneNode