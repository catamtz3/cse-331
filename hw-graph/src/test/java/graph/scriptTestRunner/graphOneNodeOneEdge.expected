# Create a graph
Created graph oneNode

#Add node to the graph
Added node a to oneNode

#Add edge to the graph
added edge aa from a to a in oneNode

# Get the nodes of the graph
oneNode contains: a[aa]